code/modules/clothing/glasses/hud.dm

```diff
@@ -171,6 +171,7 @@
 	desc = "An Ahdominian made veil that allows the user to see while obscuring their eyes. This one has an in-built security HUD."
 	icon_state = "tajblind_sec"
 	item_state = "tajblind_sec"
+	flash_protect = 1
```

Comment:
Use the define please.

Reply:
Done and done, thank you. I used the standard sechud as a reference, which also doesn't use the define